src/platforms/flutter/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -97,7 +97,7 @@ Learn more in our [User Interaction Instrumentation](/platforms/flutter/configur
 
 ### http.Client Library Instrumentation
 
-The `http.Client` instrumentation, once added the `SentryHttpClient` and enabled the `networkTracing` feature, starts a span out of the active transaction bound to the scope for each HTTP Request. The SDK sets the span `operation` to `http.client` and `description` to request `$method and $URL`; for example, `GET https://sentry.io`.
+The `http.Client` instrumentation, once added the `SentryHttpClient` and enabled the [performance](/platforms/flutter/performance/) feature, starts a span out of the active transaction bound to the scope for each HTTP Request. The SDK sets the span `operation` to `http.client` and `description` to request `$method and $URL`; for example, `GET https://sentry.io`.
```

Comment:
```suggestion
Once enabled, the `http.Client` instrumentation adds the `SentryHttpClient` and enables the [performance](/platforms/flutter/performance/) feature. This starts an active transaction span, which is bound to the scope for each HTTP Request. The SDK then sets the span `operation` to `http.client` and the `description` to request `$method and $URL`; for example, `GET https://sentry.io`
```
Same comment as above, I'm not sure I'm fully understanding this correctly. Happy to talk about it more.